backend/manager/modules/utils/src/main/java/org/ovirt/engine/core/utils/ovf/OvfOvaWriter.java

```diff
@@ -108,6 +108,7 @@ protected void writeDisk(DiskImage image) {
         _writer.writeAttributeString(OVF_PREFIX, getOvfUri(), "disk_storage_type", image.getDiskStorageType().name());
         _writer.writeAttributeString(OVF_PREFIX, getOvfUri(), "cinder_volume_type",
                 StringUtils.defaultString(image.getCinderVolumeType()));
+        _writer.writeAttributeString(OVF_PREFIX, getOvfUri(), "backup", image.getBackup().toString());
```

Comment:
so it will be "backup" with "none" or "incremental" values, right? would be better if we change this to "incremental-backup" with true/false or "backup_type" with full/incremental..

Reply:
I think taking the backup type as is (none/incremental) is easier to understand than true/false or the backup type.
that way, no other processing is needed IMHO.
wdyt?

Reply:
it is surely easier but I for one would have translated `backup=none` to "not being able to take a backup"

Reply:
Still not sure what I think about that.
@bennyz wdyt?

Reply:
I vote for incremental_backup=true/false

Reply:
ACK